### PR TITLE
crimson/osd: add tell command support

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
@ronen-fr updated and repushed.

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
jenkins test crimson perf

File: src/crimson/admin/admin_socket.cc [hunk 1]
```diff
@@ -136,25 +126,56 @@ seastar::future<> AdminSocket::finalize_response(
...
-    if (!parsed.has_value() ||
-	!validate_command(*parsed, cmdline, err)) {
-      return finalize_response(out, std::move(err));
+			      [cmd, buf=std::move(buf), this]() mutable {
+    auto maybe_parsed = parse_cmd(cmd);
+    if (auto parsed = std::get_if<parsed_command_t>(&maybe_parsed); parsed) {
+      stringstream os;
+      string desc{parsed->hook.desc};
+      if (!validate_cmd(nullptr, desc, parsed->params, os)) {
+	logger().error("{}: validation failure (incoming:{}) {}", __func__,
```
Comment (ronen-fr) [inline src/crimson/admin/admin_socket.cc]: Note __func__ inside a lambda

Comment (tchaikov) [reply src/crimson/admin/admin_socket.cc]: fixed and repushed.

**Pull request merged**